src/server/pkg/hashtree/hashtree_test.go

```diff
@@ -620,17 +621,25 @@ func TestChildIterator(t *testing.T) {
 }
 
 func TestCache(t *testing.T) {
+	// Cache with size 2
 	c, err := NewCache(2)
 	require.NoError(t, err)
 
 	h := newHashTree(t)
 	require.NoError(t, h.PutFile("foo", obj(`hash:"1d4a7"`), 1))
 	require.NoError(t, h.Hash())
+
+	// Put a hashtree into the cache
 	c.Add(1, h)
+
+	// After adding a second hashtree, the first hashtree should still be in the cache
 	c.Add(2, newHashTree(t))
 	_, err = h.Get("foo")
 	require.NoError(t, err)
+
+	// But after adding a third, the first one should be evicted
 	c.Add(3, newHashTree(t))
+	time.Sleep(time.Second) // since eviction is done concurrently, give it a second
```

Comment:
We might want this in a retry because it could potentially be flaky, although one second should be enough in general.

Reply:
We have a backoff package btw that has some retry utilities.